app/controllers/concerns/europeana/annotations_api_consumer.rb

```diff
@@ -6,20 +6,42 @@ module AnnotationsApiConsumer
     extend ActiveSupport::Concern
 
     def document_annotations(document)
-      search = nil
-      begin
-        search = Europeana::API.annotation.search(annotations_api_search_params(document))
-      rescue Europeana::API::Errors::ServerError
-        return nil
+      search_response = annotations_search_for_document(document)
+      return nil unless search_response.key?('items')
+
+      annotations_from_search_response(search_response).
+        map { |annotation| annotation_text_to_display(annotation) }.compact
+    rescue Europeana::API::Errors::ServerError
+      # @todo we may not want controller actions to fail if annotations are
```

Comment:
I also commented with Sergiu and promised that the error that he has on the annotations will be resolved as well. Thanks